README.md

```diff
@@ -1,2 +1,36 @@
-# functions-framework
-functions-frameworks overview
+# OpenFunction Functions Framework
+OpenFunction Functions Framework is designed to provide users of FaaS with a suite of tools to convert user code into application that can be adapted to multiple runtimes.
+
+## Support
+
+### Language
+
+- Go
+
+  [functions-framework-go](https://github.com/OpenFunction/functions-framework-go)
+
+### Runtime
+
+- Knative
+- Keda + Dapr
+
+### Function Type
+
+- HTTP
+
+  The framework creates a route that associates the "/" url path and the user function. After that, it starts the HTTP service according to the PORT parameter.
+
+- CloudEvent
+
+  The CloudEvent provides a handler for handling HTTP requests while binding user function. The framework creates a route that associates the "/" url path and the CloudEvent HTTP handler. After that, it starts the HTTP service according to the PORT parameter.
+
+- Data Binding
```

Comment:
Data Binding doesn't look like a function type, maybe we can call it something else?

Reply:
Call it "Bindings"?

Reply:
How about OpenFunction ?

Reply:
Ok